examples/distributed_simulation/README.md

```diff
@@ -0,0 +1,101 @@
+# Distributed Large Scale Simulation
+
+> **WARNING:**
+> **This example will consume a huge amount of tokens.**
+> **Using paid model API with this example can introduce a high cost.**
+> **Users with powerful GPUs (A800 or better) can use local inference services (such as vLLM) to run this example,**
+> **while CPU inference services such as ollama is not recommended.**
+
+This example is a large scale simulation to demonstrate the scalability of AgentScope's distributed mode. From this example, you can learn:
+
+- How to run a large number of agent servers in a GPU cluster.
```

Comment:
BTW, give an example number here, e.g. 
1. In our test, we setup xxx agents in x machines equipped with xx CPU cores. 
2. Totally xxx seconds is used with xxx agents.
